src/components/ConversationSettings/ConversationSettingsDialog.vue

```diff
@@ -30,6 +30,15 @@
 				<p v-if="recordingConsentRequired">
 					{{ t('spreed', 'The consent to be recorded will be required for each participant before joining every call.') }}
 				</p>
+				<NcCheckboxRadioSwitch v-if="supportsArchive"
+					type="switch"
+					:checked="isArchived"
+					@update:checked="toggleArchiveConversation">
+					{{ t('spreed', 'Archive conversation') }}
+				</NcCheckboxRadioSwitch>
+				<p>
+					{{ t('spreed', 'Archived conversations are hidden from the list. They will still be shown, if you have been mentioned directly.') }}
+				</p>
 				<NotificationsSettings v-if="!isGuest" :conversation="conversation" />
```

Comment:
Well, putting them next to each other highlight that notification settings need to be updated for conversation settings :p cc @nickvergessen 
![image](https://github.com/user-attachments/assets/f27d432d-a7a1-43c1-b4f2-7caf1c7c6a7b)

However, do we still need to show the toggle button as disabled as notifications setting is standard for archived conversations?

Reply:
we can set the appropriate value for notification level, but not lock it completely. In theory, if needed, I can still set `Notify about calls` after archiving, so I'll receive a notification and should also see it in the list

Reply:
> UPD: notifications are disabled by https://github.com/nextcloud/spreed/commit/e5d37ded241c6ebc27abc937bcce1fc1b943f1a0, but we can at least show it in the list, if enabed by user

We can change this. It was just my first thought. But we can change it now that we agreed that the API would toggle the other setting(s)?

Reply:
Sounds good to us, since endpoint returns updated Room object